src/QMCWaveFunctions/TrialWaveFunction.h

```diff
@@ -551,6 +551,8 @@ class TrialWaveFunction
   std::vector<std::reference_wrapper<NewTimer>> WFC_timers_;
   std::vector<RealType> myTwist;
 
+  static void checkOneParticleGradientsNaN(int iel, const GradType& grads, const std::string_view location);
```

Comment:
Add a description of this function please

Reply:
Done.